docs/developer-guide/configuration-files.md

```diff
@@ -116,6 +120,49 @@ Several configuration files (at development and / or run time) are available to
 * `new.json` Can be customized to set-up the initial new map, setting the backgrounds, initial position .. See [Maps configuration](maps-configuration.md#map-configuration) for more information.
 * `pluginsConfig.json`: Allows to configure the context editor plugins list. See [Context Editor Configuration](context-editor-config.md#configuration-of-application-context-manager) for more information.
 
+## Java System Properties
+
+It is possible to pass some specific configurations as Java system properties. These configurations regard mainly:
+
+* the location of the data directory
+* the corporate proxy settings
+
+Other configurations can be passed as Java system properties, depending on the specific needs, and they will be explained in the specific sections of the documentation that require them.
+
+### Setting Java System Properties
+
+To pass Java system properties, you can use the `-D` option when starting the Java application. For example:
+
+```sh
+java -Ddatadir.location=/path/to/data-dir -jar mapstore.war
+```
+
+In servlet containers like Tomcat, you can set Java system properties in the `setenv.sh` file (for Unix-based systems) or `setenv.bat` file (for Windows systems) located in the `bin` directory of the Tomcat installation.
+
+If you are using Tomcat as a service, you can set Java system properties in the service configuration file (for instance `/etc/default/tomcat9` in Ubuntu 20.04 ).
+
+### `datadir.location`
+
+This property allows you to externalize the location of some back-end and front-end configuration files. For more details, refer to the [Externalize Configuration](externalized-configuration.md#externalized-configuration) section.
+
+### Corporate proxy settings
+
+In case of corporate proxy, some services may require the use of an HTTP proxy to access the internet. You can pass the following Java system properties to configure the HTTP proxy:
+
+* `http.proxyHost`: specifies the HTTP proxy host.
+* `http.proxyPort`: specifies the HTTP proxy port.
+* `http.nonProxyHosts`: specifies a list of hosts that should not use the HTTP proxy.
+
+For more information, refer to the [official HTTP proxy documentation](https://docs.oracle.com/javase/8/docs/technotes/guides/net/proxies.html).
```

Comment:
```suggestion
For more information, refer to the [official HTTP proxy documentation](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/net/doc-files/net-properties.html).
```

Reply:
They are the same property, so I'd simply add the link to v 11 instead of 8 for the moment, without any specification.